app/presenters/photo_presenter.rb

```diff
@@ -4,12 +4,18 @@ def base_hash
       guid: guid,
       dimensions: {
         height: height,
-        width: width
+        width:  width
       },
       sizes: {
-        small: url(:thumb_small),
+        small:  url(:thumb_small),
         medium: url(:thumb_medium),
-        large: url(:scaled_full)
+        large:  url(:scaled_full)
+      },
+      status_message: {
+        id:             status_message.id,
+        likes_count:    status_message.likes_count,
```

Comment:
Which is only one more reason to not touch it (and maybe do a *separate* PR for putting it to use or dropping it entirely).